src/ui/findActiveFontType.js

```diff
@@ -34,13 +34,5 @@ export default function findActiveFontType(state: EditorState): string {
     return FONT_TYPE_NAME_DEFAULT;
   }
 
-  const domDoc: any = typeof document === 'undefined' ? null : document;
-
-  if (domDoc && domDoc.fonts && domDoc.fonts.check) {
```

Comment:
yay. `domDoc.fonts.check` is very unreliable (see `src/ui/canUseCSSFont.js`) and we should not use it.